fe/fe-core/src/main/java/org/apache/doris/common/util/BrokerUtil.java

```diff
@@ -114,17 +114,19 @@ public static String printBroker(String brokerName, TNetworkAddress address) {
 
     public static List<String> parseColumnsFromPath(String filePath, List<String> columnsFromPath)
             throws UserException {
-        return parseColumnsFromPath(filePath, columnsFromPath, true);
+        return parseColumnsFromPath(filePath, columnsFromPath, true, false);
     }
 
     public static List<String> parseColumnsFromPath(
             String filePath,
             List<String> columnsFromPath,
-            boolean caseSensitive)
+            boolean caseSensitive,
+            boolean isACID)
             throws UserException {
         if (columnsFromPath == null || columnsFromPath.isEmpty()) {
             return Collections.emptyList();
         }
+        int pathCount = isACID ? 3 : 2;
```

Comment:
Add comment explain `3` and `2`, better give an example

Reply:
ok